packages/axelar-wasm-std/src/msg.rs

```diff
@@ -0,0 +1,32 @@
+use cosmwasm_std::{from_json, CosmosMsg, Response, StdError, WasmMsg};
+use serde::de::DeserializeOwned;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Std(#[from] StdError),
+    #[error("no wasm msg found")]
+    NotFound,
+    #[error("multiple msgs found in the response")]
+    MultipleMsgsFound,
+}
+
+/// Get a msg wrapped inside a `WasmMsg::Execute` from a `Response`.
+/// If there are no wasm messages or more than one message in the response, this returns an error.
+pub fn from_response<T>(response: Response) -> Result<T, Error>
```

Comment:
where is this being used?

Reply:
in the integration tests (for ITS and axelarnet-gateway)